test-bench-utils/public/views/xxe.ejs

```diff
@@ -17,17 +8,19 @@
 </p>
 <% sinkData.forEach(function(sink) { %>
 <div class="row">
-  <div class="col-sm-10 col-sm-8" style="padding-bottom: 30px;">
+  <div class="col-xs-12 col-sm-6" style="padding-bottom: 30px;">
     <h4 class="sub-header">
       <code><%= sink.name %></code>
     </h4>
-    <!-- this rule is input less putting the attack vector here just for context -->
-    <textarea name="input" class="form-control" id="attack-xml"><%= input %></textarea>
     <form method="<%= sink.method %>" action="<%= sink.url %>/unsafe" target="_blank">
-      <div class="form-group"></div>
+      <div class="form-group">
+        <label>SELECT "${input}" as "test";</label>
+        <input name="input" class="form-control" value="&lt;!DOCTYPE read-fs [&lt;!ELEMENT read-fs ANY &gt;&lt;!ENTITY passwd SYSTEM &quot;file:///etc/passwd&quot; &gt;]&gt;&lt;users&gt;&lt;user&gt;&lt;read-fs&gt;&amp;passwd;&lt;/read-fs&gt;&lt;name&gt;C.K Frode&lt;/name&gt;&lt;/user&gt;&lt;/users&gt;" />
```

Comment:
it looks like there's an extra `]>`

Reply:
Not 100% sure what you mean, you're saying in the XML doc?  If you test this you'll see it still ends up rendering /etc/passwd in the response - so I think it's valid.  I html decoded and ran it w/ xmllint and it all looks good to me (xmllint says it's valid, and includes /etc/passwd when run w/ `--noent`

Reply:
got it